website/siteConfig.js

```diff
@@ -66,7 +66,10 @@ const siteConfig = {
 
     markdownPlugins: [require('./remarkableKatex'), require('./empty_thead')],
 
-    scripts: ['https://buttons.github.io/buttons.js'],
+    scripts: [
+        'https://buttons.github.io/buttons.js',
+        baseUrl + 'js/scrollspy.js',
```

Comment:
You can do `/js/scrollspy.js'.

Reply:
@edemaine Sorry, it turns out `baseUrl` is needed for scripts path. I'll open a PR.